### PR TITLE
EOS-25257: Enhance Support Bundle interfaces to take cluster.conf file path as input

### PR DESCRIPTION
**Pull request by vastradparayya**
Signed-off-by: Parayya Vastrad <parayya.vastrad@seagate.com>

# Problem Statement
- Enhance Support Bundle interfaces to take cluster.conf file path as input

# Design
-  https://jts.seagate.com/browse/EOS-25257

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: NA
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: Y
- [x] If yes for above point, Is a notification sent to all other cortx components [Y/N] NA
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by rohit-k-dwivedi — APPROVED** [review]

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/support/support_bundle_cli.py [hunk 1]
```diff
@@ -89,6 +89,8 @@ def add_args(sub_parser) -> None:
             help='Optional, Specified component support bundle will be generated')
         s_parser.add_argument('-t', '--URL', nargs='+', required=True, \
             help="Path- CORTX support bundle will be generated at specified path.")
+        s_parser.add_argument('-cp', '--cluster_conf_path', nargs='+', default='', \
```
Comment (selvakumaar5496) [inline py-utils/src/support/support_bundle_cli.py:92]: change -cp to -c

File: py-utils/src/support/support_bundle_cli.py [hunk 2]
```diff
@@ -109,16 +111,14 @@ def add_args(sub_parser) -> None:
         s_parser.set_defaults(func=SupportBundleCli.get_status)
         s_parser.add_argument('-b', '--bundle_id', nargs='+', default='', \
             help='Bundle ID of generated Support Bundle')
+        s_parser.add_argument('-cp', '--cluster_conf_path', nargs='+', default='', \
```
Comment (selvakumaar5496) [inline py-utils/src/support/support_bundle_cli.py:114]: change -cp to -c

File: py-utils/src/support/support_bundle_cli.py (hunk 1 above)
Comment (vastradparayya) [reply py-utils/src/support/support_bundle_cli.py:92]: @selvakumaar5496 we have '-c' for components. Currently we may not using but as per discussions we will be filtering using that option.

File: py-utils/src/support/support_bundle_cli.py (hunk 2 above)
Comment (sachinpunadikar) [reply py-utils/src/support/support_bundle_cli.py:114]: Lets stick to -cp for now. Need to come up with more appropriate option name

File: py-utils/src/support/support_bundle_cli.py (hunk 1 above)
Comment (sachinpunadikar) [reply py-utils/src/support/support_bundle_cli.py:92]: lets stick to -cp for now

**Pull request merged**